README.md

```diff
@@ -582,6 +584,47 @@ Otherwise, instead of raising an error, null values will be coerced as follows:
 - `string` -> `""`
 - `boolean` -> `false`
 
+<a name="largearrays"></a>
+#### Large Arrays
+
+Large arrays are, for the scope of this document, defined as arrays containing
+`20000` items or more.
+
+At some point the overhead caused by the default mechanism used by
+`fast-json-stringify` to handle arrays starts increasing exponentially, leading
+to slow overall executions.
+
+In order to improve that the user can set the `largeArrayMechanism` option with
+one of the following values:
+
+- `default` - Default behavior
+- `json-stringify` - This option will remove support for schema validation
+within **large arrays** completely. By doing so the overhead previously
+mentioned is nulled, greatly improving execution time. Mind there's no change
+in behavior for arrays with less than `20000` items
+- `array-join` - This option is a compromise between the last two.
```

Comment:
shouldn't it be the `default` one and bump a major release?

Reply:
I think this should be the default.